### PR TITLE
Fix the warnings while generating API docs

### PR DESCRIPTION
**Pull request by yongjiema**

### CODE REVIEW

File: apps/neoscan_web/lib/neoscan_web/router.ex [hunk 1]
```diff
@@ -53,12 +53,6 @@ defmodule NeoscanWeb.Router do
     get("/get_claimable/:address", ApiController, :get_claimable)
     get("/get_all_nodes", ApiController, :get_all_nodes)
 
-    get(
```
Comment (adrienmo) [inline apps/neoscan_web/lib/neoscan_web/router.ex]: this endpoint cannot be deleted because it is used by neon wallet.

Comment (yongjiema) [reply apps/neoscan_web/lib/neoscan_web/router.ex]: The removed one without required `page` parameter gets `Page not found`, does the neon wallet use the api (https://neoscan.io/docs/index.html#api-v1-get-8) with required `page` parameter mentioned in API docs, it's still included in the router.

Comment (adrienmo) [reply apps/neoscan_web/lib/neoscan_web/router.ex]: for example, neon wallet will call the page
https://neoscan.io/api/main_net/v1/get_last_transactions_by_address/AbV4wNZuC4knPDorG7cuTLnu1UdAZomUUE

to display the wallet of the address `AbV4wNZuC4knPDorG7cuTLnu1UdAZomUUE` we must make sure that after your fix, this address is still working.

Comment (yongjiema) [reply apps/neoscan_web/lib/neoscan_web/router.ex]: Oh, I see. I will update it.

**Comment by yongjiema** [comment]
Updated, please review again, thanks!

**Comment by adrienmo** [comment]
Thanks!

**Pull request merged**